dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -372,11 +372,23 @@
 			interrupts = <12 0>;
 			status = "disabled";
 			label = "ADC_1";
+			vref-mv = <3300>;
 			#io-channel-cells = <1>;
 			has-temp-channel;
 			has-vref-channel;
 		};
 
+		die_temp: dietemp {
+			label = "DIE_TEMP";
```

Comment:
Nitpick: Please put "compatible" property first. "label" is a property that is close to be obsolete and is nearly not used. As such it feels strange seeing it first.